x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -277,6 +277,7 @@ public class Constants {
         "cluster:monitor/ccr/follow_info",
         "cluster:monitor/ccr/follow_stats",
         "cluster:monitor/ccr/stats",
+        "cluster:monitor/xpack/usage/data_lifecycle",
```

Comment:
I think we need feature flag check here or it will fail at release build CI.
```suggestion
        DataLifecycle.isEnabled() ? "cluster:monitor/xpack/usage/data_lifecycle" : null,
```

Reply:
Thank you! Missed it